src/helpers/utils.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Combine name and title of provider
+ * @param {String} name A provider name.
+ * @param {String} title A provider title.
+ * @return {String} Name with the title.
+ */
+export function formatProviderHeading(name, title) {
+  return `${name}, ${title}`;
+}
+
+/**
+ * Format location options for dropdown list
+ * @param {[{}]} locations An array of available locations.
+ * @return {[]} An array of locations formatted for dropdown use.
+ */
+export function formatLocationsForDropdown(locations) {
+  return locations.map((location) => ({
+    value: location.name,
+    label: location.abbrev,
+  }));
+}
+
+/**
+ * Format and pluralize providers text
+ * @param {Number} amountOfProviders Number of providers.
+ * @param {String} location A selected location.
+ * @return {String} Formatted string with correct pluralization.
+ */
+export function pluralizeProviders(amountOfProviders, location) {
+  if (amountOfProviders === 1) {
```

Comment:
Good attention to detail here. One improvement could be making another case where there are zero providers and maybe returning some 'try again later' message. Right now it's just an empty list which isn't great UX